_posts/2024-01-11-gsoc2023_xhgchen.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: GSoC 2023 - Add calculations for self-diffusivity and viscosity
+---
+
+**Contributor**: Xu Hong Chen
+
+**Mentors**: Hugo MacDermott-Opeskin ([@hmacdope](https://github.com/hmacdope)), Orion Cohen ([@orionarcher](https://github.com/orionarcher))
+
+**Organization**: [MDAnalysis](https://www.mdanalysis.org/)
+
+Transport property analyses are among [the most requested](https://github.com/MDAnalysis/mdanalysis/issues/2187) features in the [MDAnalysis project](https://www.mdanalysis.org/). Biomolecular researchers and chemical engineers routinely use these measurable values describing the transfer of mass, momentum, heat, and charge in their regular work.[^1]<sup>,</sup> [^2] [My Google Summer of Code (GSoC) project](https://summerofcode.withgoogle.com/programs/2023/projects/4vt9npUg) aimed to add methods for calculating self-diffusivity and viscosity, the two most common transport properties in the literature,[^1] to MDAnalysis with a focus on user-friendliness and test-driven development. Initially, I proposed to:
+
+- Implement a class to calculate self-diffusivity via the Green-Kubo method
+- Write a utility function to calculate shear viscosity via the Einstein method
+- Create a utility function to calculate shear viscosity via the Green-Kubo method
+- Test and document the new features to ensure they are reliable and easy to use
+
+After some discussion that began with Discord user jenclark's suggestion to try the Helfand method for viscosity to better integrate the feature wtih MDAnalysis, we shifted our priorities away from the utility functions to focus on a new Einstein-Helfand viscosity class. That extra research and trial and error were well worth the result - better software with a more simple and robust API.

```

Comment:
The handle is almost certainly NOT `jenclark` but @jaclark5 — please check.

Reply:
Discord handle was jenclark, but have changed to GH handle for consistency and easier connection to @jaclark5's profile